docs/details/blocks/General/Sub.md

```diff
@@ -0,0 +1,11 @@
+In Chainblocks a block receives an input, processes it, and produces an output. Usually this output is different from the input that the block processed. 
+
+However, in some cases, we want the block to return the same value as its output that it reveived as an input (for example when we want the next block in the sequence to process the same input that the previous block processed).
+
+There is a pass-through parameter available for some blocks (like `Match`) that can allow such blocks to pass-through their own input as their output. To achieve this same effect with other blocks, we can wrap these blocks in the `Sub` block container (also called the `Sub` flow).
+ 
+The `Sub` flow outputs the same value that it receives as input. It also passes this input value to the blocks within it for their processing, but doesn't use their generated output. The output of a `Sub` block is always what came in as its input.
+
+So this method gives a psuedo pass-through capabilty to any block (if you wrap just that one block with a `Sub` block) or to any sequence of blocks (if you wrap a whole sequence of blocks with a `Sub` block).
+
+By nesting `Sub` blocks you can simulate even more flexible and powerful block execution paths.
```

Comment:
I would add a remark that a shortcut exists: `|`. e.g.

```clj
.input
(Sub ;; Sub1: a set of block sequences
  ;:Blocks
  (->
    ;; some other blocks here
  ))
(Sub ;; Sub2: a set of block sequences
  ;:Blocks
  (->
    ;; some other blocks here
  ))
;; equivalent
.input
(| ;; some other blocks here without requiring (->)
)
(| ;; some other blocks here without requiring (->)
)
```

Reply:
Added this as a `!!! note`.